views/DataByAgeDistribution/SelectFeature.html

```diff
@@ -25,5 +25,77 @@
             </div>
         </div>
     </form>
+	{% if data and TypeOfTable == 1%}
+		<h2 class="text-center display-5 mt-5">Number of confirmed, deaths, and recovered for each country for age </h2>
+		{% if AgeRange == 1%}
```

Comment:
I think you could have one header (h2) and inside of it add your if-elseif logic. 
<h2> 
        I like
        {% if false %}  
               Tacos 
        {% elseif true %}
               Burritos
          {% else %}
                Churros
           {% endif %}
</h2>

Something like this. Haven't tested it, but I think it will work!